conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
```

Comment:
why do we a have elear-develop??

Reply:
that's the naming convention for develop branches which are for open source repos